src/main/java/org/example/models/JobRole.java

```diff
@@ -0,0 +1,124 @@
+package org.example.models;
+
+import java.util.Date;
+
+public class JobRole {
+    private int roleId;
+    private String roleName;
+    private String description;
+    private String responsibilities;
+    public String locations;
+    private String linkToJobSpec;
+    private String capability;
```

Comment:
Shouldn't this be an enum as it's an enum in the database

Reply:
Your database design has enums so it should match your code no matter what the database returns